guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p>costpath_as_polyline is not intended properly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Fixed it.</p>